awx/ui_next/src/screens/User/shared/UserForm.jsx

```diff
@@ -11,7 +11,7 @@ import FormField, {
   FormSubmitError,
 } from '../../../components/FormField';
 import OrganizationLookup from '../../../components/Lookup/OrganizationLookup';
-import { required, requiredEmail } from '../../../util/validators';
```

Comment:
Is there some validator that's "not required, but if you fill it in, we'll yell if it doesn't look email-like" ?

Reply:
From what I can see, this PR does that:

![useremail](https://user-images.githubusercontent.com/9753817/108778422-5c76bf00-7533-11eb-9d38-60831fbae286.gif)

Reply:
^ Actually, that message is coming from the server. It's just so darn fast I couldn't tell the difference :sweat_smile:

Reply:
... Anyway, if you wanted to do something like this (with a client-side validator to avoid a request) you _totally_ could.

You could do it by copying the `requiredEmail` validator, renaming it, and removing this line: https://github.com/ansible/awx/blob/devel/awx/ui_next/src/util/validators.jsx#L50-L52

Reply:
Eh, I think it's fine if it's enforced on the API side.